spec/controllers/job_messages_controller_spec.rb

```diff
@@ -74,12 +74,13 @@
       end
     end
   end
-  context 'as an admin' do
-    let(:user) { create :admin }
-    it_behaves_like 'job messages'
+  describe 'When user is an authenticated admin' do
+    let!(:user) { create(:admin) }
```

Comment:
Is there a reason you're using `let!` instead of `let` (here and in a bunch of other places)? It seems like in most cases `let` works just as well. And we prefer to use `let` whenever possible because it lazily creates objects which means it's more efficient in cases where the object is not required.